.changeset/neat-glasses-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-import': patch
```

Comment:
Think this needs to be a `minor` change as it's breaking. We need to pass `t` into `generateDefaultStepper` now.

Probably want to add some instructions on how to do that too for consumers of the method.

Reply:
I added some change document in changeset, please help to review or reconstruct.

Reply:
I added some change document in changeset, please help to review or reconstruct.